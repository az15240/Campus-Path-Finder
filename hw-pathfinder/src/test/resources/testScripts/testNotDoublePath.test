# Tests the special case in which the input string for edge label is not a double

CreateGraph g

AddNode g A
AddNode g B
AddNode g C

AddEdge g A B 1.23456
AddEdge g B A 2.231941475257139652193561
AddEdge g A C not-a-double

ListNodes g

ListChildren g A

FindPath g A C
FindPath g C A