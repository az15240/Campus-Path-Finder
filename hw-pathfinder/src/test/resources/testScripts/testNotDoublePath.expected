# Tests the special case in which the input string for edge label is not a double

created graph g

added node A to g
added node B to g
added node C to g

added edge 1.235 from A to B in g
added edge 2.232 from B to A in g
Provided label is not a double!

g contains: A B C

the children of A in g are: B(1.235)

path from A to C:
no path found
path from C to A:
no path found